sdk/dotnet/CloudBuild/Inputs/TriggerBuildArtifactsObjectsArgs.cs

```diff
@@ -32,13 +32,19 @@ public InputList<string> Paths
             set => _paths = value;
         }
 
+        [Input("timings")]
+        private InputList<Inputs.TriggerBuildArtifactsObjectsTimingArgs>? _timings;
+
         /// <summary>
         /// -
         /// Output only. Stores timing information for pushing all artifact objects.
         /// Structure is documented below.
         /// </summary>
-        [Input("timing")]
-        public Input<Inputs.TriggerBuildArtifactsObjectsTimingArgs>? Timing { get; set; }
+        public InputList<Inputs.TriggerBuildArtifactsObjectsTimingArgs> Timings
```

Comment:
This (along with many other similar changes in this PR) is a breaking change. Are we bumping the major version before release? Or do we need to revert this back to how it was previously?

Reply:
There was no plans to bump the version on this - this is actually what changed upstream in this minor release. We might be able to use MaxItems to remediate if the API doesn't actually take more than 1 item in this case

There are other times we've accepted a breaking change to these providers when things like this happen

//cc @leezen

Reply:
Yeah, agrees with Justin here, the changes are quite significant. Are they all `MaxItems=1`-able?